.eslintrc.js

```diff
@@ -149,12 +149,7 @@ module.exports = {
         'no-await-in-loop': 'warn',
         'no-compare-neg-zero': 'error',
         'no-cond-assign': ['error', 'always'],
-        'no-console': [
-            'error',
-            {
-                allow: ['warn', 'error', 'info'],
-            },
-        ],
+        'no-console': 0,
```

Comment:
prefer `'off'` (I can never remember is `0` = `off` or `error`)

Reply:
Also if it's off, maybe we can just remove the line

Reply:
Did it in https://github.com/DataDog/build-plugin/pull/65